app/assets/javascripts/my_modules/results.js

```diff
@@ -217,14 +189,40 @@
       e.preventDefault();
       e.stopPropagation();
       e.stopImmediatePropagation();
-      var el = $(element);
-      if(confirm(el.data('confirm-text'))) {
+      let el = $(element);
+      if (confirm(el.data('confirm-text'))) {
         animateSpinner();
         $('#' + el.data('form-id')).submit();
       }
     }
 
-    var publicAPI = Object.freeze({
+    function init() {
+      initHandsOnTables($(document));
+      expandAllResults();
+      applyCollapseLinkCallBack();
+      applyCreateWopiFileCallback();
+
+      $(function() {
+        $('#results-collapse-btn').click(function() {
+          $('.result .panel-collapse').collapse('hide');
+          $(document).find('span.collapse-result-icon').each(function() {
+            $(this).addClass('fa-caret-square-down');
+            $(this).removeClass('fa-caret-square-up');
+          });
```

Comment:
You can make it without cycle:
`$(document).find('span.collapse-result-icon').addClass('fa-caret-square-down').removeClass('fa-caret-square-up')`